game/world/managers/CommandManager.py

```diff
@@ -876,6 +876,17 @@ def pwdchange(world_session, args):
             return -1, 'please use it like: .pwdchange current_password new_password new_password'
 
 
+    @staticmethod
+    def save_location(world_session, args):
+        if args:
+            logline = f'{world_session.player_mgr.location.x} {world_session.player_mgr.location.y} {world_session.player_mgr.location.z} {world_session.player_mgr.map_id} - {args}'
+            f = open(os.path.join(os.getcwd(), "locations.log"), "a")
```

Comment:
I think it will be a good idea to create a config variable inside the existing Logging for these kind of paths. For example, name it `log_dev_path` where this `locations.log` can be saved (and maybe more dev / debugging logs in the future). Also make sure to create it if it doesn't exist (check first line of `ChatLogManager.process_logs` method to check how to do it).

Reply:
Problem: this doesn't work on Windows.

Reply:
What doesn't work?

Reply:
Never mind - it does, just not as practical as I hoped.